src/app/series/series.component.css

```diff
@@ -0,0 +1,7 @@
+/*
+ * Series edit styling
+ */
+form {
+  padding: 20px 0;
+  max-width: 800px;
+}
```

Comment:
Still pretty new to this project, so I'm just asking here. Seeing custom styles for components that seem fairly general on the surface doesn't feel so good. I'm wondering what the eventual roadmap looks like for styling the app. Does it involve an external style guide at all that could be pulled into our various Angular projects?